framework-cloud/framework-openshift-templates/src/main/java/org/kie/cloud/openshift/scenario/builder/ImmutableKieServerScenarioBuilderImpl.java

```diff
@@ -64,30 +66,36 @@ public ImmutableKieServerScenarioBuilder withContainerDeployment(String kieConta
     }
 
     @Override
-    public ImmutableKieServerScenarioBuilder withSourceLocation(String gitRepoUrl, String gitReference, String gitContextDir) {
-        envVariables.put(OpenShiftTemplateConstants.SOURCE_REPOSITORY_URL, gitRepoUrl);
-        envVariables.put(OpenShiftTemplateConstants.SOURCE_REPOSITORY_REF, gitReference);
-        envVariables.put(OpenShiftTemplateConstants.CONTEXT_DIR, gitContextDir);
+    public ImmutableKieServerScenarioBuilder withSourceLocation(String gitReference, String gitContextDir) {
```

Comment:
The gitReference and gitContextDir doesn't seem to be used here? Did you accidentally deleted env varible configuration from above?

Reply:
Thanks for spotting this!
Fixed